odfuzz/generators.py

```diff
@@ -11,7 +11,11 @@
 from odfuzz.config import Config
 
 START_DATE = datetime.datetime(1970, 1, 1, 0, 0, 0)
-END_DATE = datetime.datetime(9999, 12, 31, 23, 59, 59)
+END_DATE = datetime.datetime(3000, 12, 31, 23, 59, 59)
```

Comment:
9999 is valid SAP input sadly.. 

But we can go in a way that the START-END range will be 1970 - 3000 + the timestamp value for 9999-12-31-23-59-59 (= "Infinity" in SAP world, for example validity period end) will be hardcoded String.. it is in the end used in GET/POST string anyway.

Then the generator do not need to tackle real DateTime object with such high value and go OK on LInux and fail on Windows.

Reply:
Implemented.